### ISSUE TITLE
How did you handle batch norm?

### ISSUE THREAD

**Issue by nirandaperera** [kakaobrain/torchgpipe#14]
Hi, 

Can you guys explain how you managed to do the batchnorm operation when you are dealing with microbatches?

Best 
Niranda

**Comment by sublee** [comment]
It's explained in https://torchgpipe.readthedocs.io/en/stable/gpipe.html#deferred-batch-normalization.

**Comment by nirandaperera** [comment]
Thanks. My bad!
